ext/ffi/ffi.c

```diff
@@ -886,6 +886,11 @@ static void zend_ffi_callback_trampoline(ffi_cif* cif, void* ret, void** args, v
 	}
 	free_alloca(fci.params, use_heap);
 
+	if (EG(exception)) {
+		zend_throw_error(zend_ffi_exception_ce, "Uncaught %s in PHP FFI callback", ZSTR_VAL(EG(exception)->ce->name));
```

Comment:
Maybe make the message something like `"Throwing from FFI callbacks is not allowed"`? The uncaught exception is already printed anyway.

Reply:
Yes, makes sense.